src/PIL/ImageFont.py

```diff
@@ -139,6 +139,8 @@ def getsize(self, text, *args, **kwargs):
         """
         .. deprecated:: 9.2.0
 
+        See :ref:`deprecations <Font size and offset methods>` for more information.
+
         Use :py:meth:`.getbbox` or :py:meth:`.getlength` instead.
```

Comment:
Oh and let's flip the order:

> _Deprecated since version 9.2.0._
>
> Use x instead.
>
> See [deprecations](https://pillow--6556.org.readthedocs.build/en/6556/deprecations.html#font-size-and-offset-methods) for more information.



```suggestion
        Use :py:meth:`.getbbox` or :py:meth:`.getlength` instead.

        See :ref:`deprecations <Font size and offset methods>` for more information.
```